### PR TITLE
Add Route53 DNS provider

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)

### CODE REVIEW

File: guides/common/modules/proc_configuring-dns-route53.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+----
+# {foreman-installer} \
+--enable-foreman-proxy-plugin-dns-route53 \
+--foreman-proxy-dns true \
+--foreman-proxy-dns-provider route53 \
+--foreman-proxy-plugin-dns-route53-aws-access-key _My_AWS_Access_Key_ \
+--foreman-proxy-plugin-dns-route53-aws-secret-key _My_AWS_Secret_Key_
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*.
+. Select your {SmartProxy} and from the *Actions* list, select *Refresh*.
```
Comment (ekohl) [inline guides/common/modules/proc_configuring-dns-route53.adoc:20]: The installer should take care of the refresh.

Comment (maximiliankolb) [reply guides/common/modules/proc_configuring-dns-route53.adoc]: Thanks, removed.

**Comment by AmosZh** [comment]
This is just what has been missing. Thank you @maximiliankolb

**Review by ekohl — APPROVED** [review]

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   b17c1e1b1f..acbc826cfd  3.10 -> 3.10
   2dbbb59d6c..45df2538aa  3.9 -> 3.9

**Pull request merged**